ceagle/api/v1/status.py

```diff
@@ -29,9 +32,17 @@ def get_status(period):
 
 
 @bp_status.route("/health/<period>")
-@fake_status.get_status_health
 def get_status_health(period):
-    return flask.jsonify("fixme!")
+    health_client = client.get_client("health")
+    if not health_client:
+        return flask.make_response(
+            json.dumps({"error": "No health endpoint configured"}),
+            500,
+        )
+
+    api_endpoint = "/api/v1/health"
```

Comment:
Endpoint comes from config, do not hard-code such data
https://github.com/seecloud/ceagle/blob/297b03a51ed0108b60088c5626ad7d0f25cb3299/etc/eagle_config.json#L10